webapp-django/crashstats/crashstats/models.py

```diff
@@ -87,6 +87,20 @@ class Meta:
         unique_together = ('vendor_hex', 'adapter_hex')
 
 
+class Signature(models.Model):
+    """Bookkeeping table to keep track of when we first saw a signature"""
+    signature = models.TextField(
+        unique=True,
+        help_text='the crash report signature'
+    )
+    first_build = models.BigIntegerField(
+        help_text='the first build id this signature was seen in'
+    )
+    first_date = models.DateTimeField(
```

Comment:
This was "first_report" in the old table, but I'm renaming it to "first_date". "report" is ambiguous and confusing.